backend/printmach.ml

```diff
@@ -23,34 +23,47 @@ open Interval
 
 module V = Backend_var
 
+let loc ?(wrap_out = fun ppf f -> f ppf) ~reg_class ~unknown ppf l = 
+  match l with 
+  | Unknown -> unknown ppf
+  | Reg r ->
+      wrap_out ppf (fun ppf -> fprintf ppf "%s" (Proc.register_name r))
+  | Stack(Local s) ->
+      wrap_out ppf (fun ppf ->
+        fprintf ppf "s%s%i"
+          (match reg_class with
+          | 0 -> ""
+          | 1 -> "f"
+          | c -> Printf.sprintf "Unknown(%d)" c)
+          s)
```

Comment:
This looks tied to the amd64 backend.

Reply:
Well, no it's not and is OK with the two backends
in the repository. It would be slightly better to
define _e.g._ Proc.register_class_xyz : int -> string`
to get a prefix or a description.

Reply:
`Proc.register_class_xyz` is an actual proposition or the `xyz` is just a placeholder?
Also how would you ensure it doesn't collide with the "si" and "so" for `Incoming` and `Outgoing`?
We could rename those into "pi" "po" as "parameters incoming" and "parameters outgoing".

Reply:
I was just too lazy to come up with a proper name...
the format could be `sl%s` with the `%s` coming
from that `Proc` function. That way there is no
possible collision (it is also a bit more uniform
"sl"/"si"/"so"/"ds" as prefixes).

Reply:
By the way, `Proc` is one of these backend-specific
modules: `proc.mli` is shared between all backend,
while there is one `proc.ml` per backend.